lxd/certificates.go

```diff
@@ -716,6 +717,11 @@ func certificateGet(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
+	err = s.Authorizer.CheckPermission(r.Context(), r, entity.CertificateURL(cert.Fingerprint), auth.EntitlementCanView)
```

Comment:
Do we have a test for this to check that non-admin users can only update their cert?

Reply:
Yes it's called `test_certificate_edit` and is in `tls_restrictions.sh`